features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Documents/Details.jsx

```diff
@@ -119,9 +120,21 @@ class Details extends React.Component {
      * @memberof Details
      */
     componentDidMount() {
-        const { apiUUID, documentId } = this.props.match.params;
-        const api = new Api();
-        const promisedDocument = api.getDocument(apiUUID, documentId);
+        const { apiType } = this.props.api;
```

Comment:
this might throw an eslint error, Its better to change it as 
`const {api} = this.props`

and then use api.apiType.

Similarly thing is seen inside the switch case

Reply:
fixed with https://github.com/wso2/carbon-apimgt/pull/6759/commits/0c218762d38e7db55b9b2b6306a4a1027e6d4cb0